db/system_keyspace.cc

```diff
@@ -2668,10 +2668,16 @@ future<std::optional<mutation>> system_keyspace::get_group0_schema_version() {
     return get_scylla_local_mutation(_db, "group0_schema_version");
 }
 
-static constexpr auto SERVICE_LEVELS_VERSION_KEY = "service_level_version";
+future<system_auth_keyspace::version_t> system_keyspace::get_auth_version() {
+    auto str_opt = co_await get_scylla_local_param("auth_version");
+    if (!str_opt) {
+        co_return system_auth_keyspace::V1;
+    }
+    co_return system_auth_keyspace::version_t(*str_opt);
```

Comment:
`using version_t = std::string_view;`

Oops, you are returning a `std::string_view` which refers to a stack variable (`std::optional<sstring> str_opt`).

Reply:
thanks